be/src/formats/parquet/metadata.h

```diff
@@ -87,6 +87,11 @@ class FileMetaData {
 
     const ApplicationVersion& writer_version() const { return _writer_version; }
 
+    size_t estimate_memory() const {
+        size_t version_size = _writer_version.application_.length() + _writer_version.build_.length();
+        return sizeof(tparquet::FileMetaData) + sizeof(uint64_t) + _schema.estimate_memory() + version_size;
```

Comment:
Is using `sizeof` accurate? On-heap memory is not counted

Reply:
> Is using `sizeof` accurate? On-heap memory is not counted

Not accurate, and in fact it is difficult to calculate the accurate memory of this class objects. So we add some comments in the related functions. In the other hand, this value is only used for simple memory estimation and does not need to be completely accurate.